ui/client/cypress/fixtures/testProcess2.json

```diff
@@ -0,0 +1,287 @@
+{
```

Comment:
I think we'd better have simpler, smaller diagram here (e.g. using mostly built-in nodes like filter). Now it's unnecessarily brittle - if we change implementation of our demo nodes, this test will break, as the process will be no longer valid